### DISCUSSION TITLE
Include jargon words in topics

### DISCUSSION THREAD

**Discussion by Nee-DS** [MaartenGr/BERTopic#1166] [Q&A] [answered]
Can we include a list of customized jargon words to the model and ask to give weightage to it while constructing topics or training?

**Comment by MaartenGr** [comment, marked as answer]
You can use guided topic modeling to give more weight to certain words by providing them as words in a seeded topic list. This will also influence the topic creation but it should not be major. 

You can find more about that [here](https://maartengr.github.io/BERTopic/getting_started/guided/guided.html).
